### ISSUE TITLE
Stellarium Vade Mecum

### ISSUE THREAD

**Issue by EricMyers47** [Stellarium/stellarium#3488]
I wanted to let you know that I have created a "Vade Mecum"  (a pocket "cheat sheet") for Stellarium.   http://www.spy-hill.net/myers/astro/stellarium/StellariumVadeMecum.pdf

As you will see, it's marked to be folded up to put in your pocket (Letter size paper - I'm not sure if it will work with A4). The content is arranged so that the most common functions are documented first, then you open a fold to get more advanced functions, then open another fold to get to even more advanced topics.

As you will also see, the license is CC-by-nc-sa with the idea that anyone can use and share it.

And of course suggestions for improvements are welcomed, especially for correcting any errors.

**Comment by 10110111** [comment]
How did you create it? Just manually in a word processor?
Also, why are some items crossed out (in Oculars View page)?

**Comment by EricMyers47** [comment]
I created it in PowerPoint.   I've done other things with the same folding
pattern so I had a template.

The oculars are crossed out because they don't work for me as advertised.
I suppose I should submit a ticket,
but as you can see this is my first.

-Eric

**Comment by ilkant** [comment]
If there is source code (.ppt) I could localize it. :)

**Comment by EricMyers47** [comment]
Good idea.   And maybe "port"  it to A4 as well?   (I don't have any A4
paper here but if someone does and wants to try it I'd like to know how it
looks.   Probably just too much empty space at the bottom edges, which is
not horrible.)

I'm doing a training session this month ("Stellarium 101") so I looked at
this again (and finally remembered to let you folks know about this
document) and now I want to rearrange it a bit.  Let me do that first, and
then I can give you the .ppt.     Also, constructive criticism is
welcomed.  The design principle is simplest first, more complex deeper
"inside".   Which is why I now want to move the mouse and menus page to the
back cover and put the time controls inside.

On an unrelated topic (but I didn't know where to ask this) in the past
I've created a few landscapes and I had credentials to add them to the
list, but I seem to have forgotten how to do it or the password and I have
a few more new ones.
See http://www.spy-hill.net/myers/astro/stellarium/

Regards,
Eric

**Comment by ilkant** [comment]
I printed that PDF double sided to one A4 paper. I think that it would be also nice to A5 paper.

And I think that if we think localization, it would be nice if that is in Transifex. And it can be document in Stellarium installation package. These are my thoughts.

**Comment by 10110111** [comment]
Does Transifex support PPT?

**Comment by ilkant** [comment]
> Does Transifex support PPT?

I don't know. I meant that the translation would be done in Transifex. Then some computer program would turn it into a PDF file. I don't know if there are any already. You can always think of other file formats such as .doc, .xhtml (+css), etc.

**Comment by EricMyers47** [comment]
Updated version in PowerPoint is at  [http://www.spy-hill.net/myers/astro/stellarium/StellariumVadeMecum.pptx](http://www.spy-hill.net/myers/astro/stellarium/StellariumVadeMecum.pptx).

**Comment by ilkant** [comment]
> Updated version in PowerPoint is at http://www.spy-hill.net/myers/astro/stellarium/StellariumVadeMecum.pptx.


I use LibreOffice and it's Impress. This shows some of the text outside the paper. You can publish pdf also. With information technology, it often happens that a product is used in many different environments. And something like that can have unwanted effects.
